pkg/filetree/filetree.go

```diff
@@ -43,11 +43,20 @@ func (t *FileTree) Copy() (*FileTree, error) {
 }
 
 // AllFiles returns all files and directories within the FileTree.
-func (t *FileTree) AllFiles() []file.Reference {
+func (t *FileTree) AllFiles(types ...file.Type) []file.Reference {
```

Comment:
It would be nice to document type filtering behavior and the "default" behavior (no supplied types) in the doc comment.